www/src/pages/docs/40_StoreCreation.md

```diff
@@ -0,0 +1,728 @@
+passed new object of the same type. But an action can be any type and is not limited to anything.
+
+In our case the type `T` of the store is `List<Person>`, but as action it is sufficient to have a single 
+`Person`-object. 
+
+```kotlin
+val storedPersons = object : RootStore<List<Person>>(emptyList()) {
+
+    val addPerson: Handler<Person> = handle { persons, newPerson ->
+        //                 ^^^^^^^            ^^^^^^^  ^^^^^^^^^
+        //                 defines the        the      the new
+        //                 type of parameter  "old"    person
+        //                 to pass            state    to add
+        if (persons.any { it.id == newPerson.id }) persons else persons + newPerson
+        //  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^               ^^^^^^^^^^^^^^^^^^^
+        //  use the "old" state and the action                  add the action to the
+        //  to check for duplicates                             old state to create the new state
+    }
+}
+```
+The above handle-code shows some typical pattern: We create the new store's state by analyzing first the "old" state
+with some information from the action and then decide whether the old state could remain or there must be some update
+also using some information from the action.
+
+Let's "simulate" some UI, that uses this handler:
+
+```kotlin
+val storedPersons = object : RootStore<List<Person>>(emptyList()) {
+    val addPerson: Handler<Person> = handle { persons, newPerson ->
+        if (persons.any { it.id == newPerson.id }) persons else persons + newPerson
+    }
+}
+
+render {
+// (rendering of person omitted) 
+
+    section {
+        button {
+            +"Add Fritz"
+            clicks.map {
+                // we define some static person; in real life this might be created by some user input
+                Person(1, "Fritz", setOf(Interest.Programming, Interest.History))
+            } handledBy storedPersons.addPerson
+            // in order to call `addPerson`, we need a `Flow<A>` as first parameter for `handledBy`
+            // So the flow defines the action `A`, that the handle-code will receive,
+            // in this case some `Person`-object
+        }
+    }
+}
+```
+
+#### Use Meta-Information in Action
+
+In order to implement the second requirement, which is to add some interest to some specific person, we should
+recap the shape of our model: `Person` is an *entity*, so it has some stable identifier. To determine the person in
+the list, which some new interest should be added to this person's interests list, we simply can rely on the
+`Person.id`-property.
+
+So our action consists of two parts:
+- the id of the specific person, where the interest will be added to (meta-information)
+- the interest to add (information)
+
+For simple cases like this, a `Pair` is a sufficient choice to group both information. But of course you could also
+create some (data) class or any other kotlin feature that fits.
```

Comment:
Wouldn't you do this by deriving a Store for this particular Person and than adding the Interes there?

Reply:
The documentation has not yet arrive to those topics; so we can choose between changing examples or to accept solutions, that can be solved otherways too (we could place some hint box that mentions another approach?)